### ISSUE TITLE
Myrmexes piling up in caves

### ISSUE THREAD

**Issue by WBgroup** [Alex-the-666/Ice_and_Fire#1236]
Myrmexes are piling up in small cave systems without the ability to leave it.
This might be a thread for servers and clients.
I have quark installed so stalagmites and stalactites are added, but there was none in that place.
![2019-01-01_14 58 39](https://user-images.githubusercontent.com/25838025/50573618-d7227480-0dd7-11e9-94ae-ae866221ae66.png)

**Comment by scotty9090** [comment]
Seeing the same issue.  /forge entities list is showing me upwards of 80 (of a given type).  I checked this because my TPS was tanking for no apparent reason.  After running several /kill commands my TPS returned to normal.  I don't see any config setting to control the spawn chance directly other than the chance to spawn nests (which I've changed to zero).  Unfortunately, now that a lot of world gen is done I'm not sure this will matter.  IIRC, I have also changed the rate that a queen hatches larva to either disable it or be as slow as possible.  Anyway to disable new spawns completely?
